app/frontend/src/components/admin/StatusTable.vue

```diff
@@ -0,0 +1,147 @@
+        :expanded.sync="expanded"
+        item-key="inspectionStatusId"
+        class="status-table"
+      >
+        <template v-slot:item.createdAt="{ item }">{{ formatDate(item.createdAt) }}</template>
+
+        <template v-slot:item.grade="{ item }">
+          <v-icon v-if="item.grade && item.grade.toUpperCase() === 'PASS'" color="green">check</v-icon>
+          <v-icon
+            v-else-if="item.grade && item.grade.toUpperCase() === 'FAIL'"
+            color="secondary"
+          >close</v-icon>
+          <v-icon v-else color="secondary">mdi-help-circle</v-icon>
+        </template>
+
+        <template v-slot:item.inspectorName="{ item }">{{ item.inspectorName }}</template>
+
+        <template v-slot:item.inspectionDate="{ item }">{{ formatDate(item.inspectionDate) }}</template>
+
+        <template v-slot:expanded-item="{ headers, item }">
+          <td :colspan="headers.length">
+            <p><strong>Decision Reason</strong>{{ item.reasonsForDecision }}</p>
+            <p><strong>Guidance Plan</strong>{{ item.guidancePlan }}</p>
+          </td>
+        </template>
+      </v-data-table>
+    </v-container>
+  </v-card>
+</template>
+
+<script>
+import ipcService from '@/services/ipcService';
+
+export default {
+  name: 'StatusTable',
+  computed: {
+    responsiveCell() {
+      return this.$vuetify.breakpoint.name == 'xs'
+        ? 'v-data-table__mobile-table-row'
+        : '';
+    }
+  },
+  data() {
+    return {
+      headers: [
+        { text: 'Status', value: 'status' },
+        { text: 'Date', align: 'start', value: 'createdAt' },
+        { text: 'Grade', align: 'start', value: 'grade' },
+        { text: 'Assigned Inspector', value: 'inspectorName' },
+        { text: 'Inspection Date', value: 'inspectionDate' }
+      ],
+      expanded: [],
+      statuses: [],
+      ipcPlanId: this.$route.params.ipcPlanId,
+      loading: true,
+      showAlert: false,
+      alertType: null,
+      alertMessage: ''
+    };
+  },
+  methods: {
+    formatDate(date) {
+      return date ? new Date(date).toLocaleString() : '';
+    },
+    getData() {
+      ipcService
+        .getIPCInspectionStatuses(this.ipcPlanId)
+        .then(response => {
+          this.statuses = response.data;
+          if (!this.statuses.length) {
+            this.showTableAlert('warning', 'No inspection statuses found');
+          }
+        })
+        .catch(error => {
+          this.showTableAlert('error', error.message);
+        })
+        .finally(() => {
+          this.loading = false;
+        });
+    },
+    showTableAlert(typ, msg) {
+      this.showAlert = true;
+      this.alertType = typ;
+      this.alertMessage = msg;
+      this.loading = false;
+    }
+  },
+  mounted() {
+    this.getData();
+  }
+};
+</script>
+
+<style scoped>
+.status-search {
+  width: 100%;
+  max-width: 20em;
+  float: right;
+}
+.status-table {
+  clear: both;
+}
+.status-table >>> tr.v-data-table__expanded__row td {
```

Comment:
All these styles from here down to line 146 are only required if we're going to show the expanded row in the table